src/components/Search/Search.js

```diff
@@ -12,11 +12,16 @@ const Search = ({ query }) => {
   const [loading, setLoading] = useState(false)
   const [error, setError] = useState()
   const router = useRouter()
+  const patt = /[0-9]{1}[0-9]{7}/g
```

Comment:
Please could we use a more specific name like `workOrderReferenceRegex`.

Does `/[0-9]{8}/g` give us the same result with simpler code?

Reply:
Thanks for pointing this out.
First I had `/[0-9]{8}/g` to match any 8 digit number but then I thought of change it to `/[1]{1}[0-9]{7}/g` to match the current work number ref. Although it seems I lost it on a stash 😂 
I will change it to `/[0-9]{8}/g`so it match any 8 digit number.